docs/Guides/Ecosystem.md

```diff
@@ -538,6 +538,8 @@ middlewares into Fastify plugins
   OSM plugin to run overpass queries by OpenStreetMap.
 - [`fastify-override`](https://github.com/matthyk/fastify-override)
   Fastify plugin to override decorators, plugins and hooks for testing purposes
+- [`fastify-passkit-webservice`](https://github.com/alexandercerutti/fastify-passkit-webservice)
```

Comment:
It should be possible to run the test automatically by adding this file to the repo:
https://github.com/fastify/fastify-url-data/blob/master/.github/workflows/ci.yml

Reply:
I think I can do that. Otherwise I'll create one ad-hoc. Is it mandatory?

Reply:
Seems not compatible with `pnpm`, so I cannot use it.
I guess I just need to implement things manually.

Is it mandatory?

Reply:
> Seems not compatible with pnpm

Do you have an error to show?
Any JS project with a valid package.json should work

> Is it mandatory?

It is not a written rule, but we try to improve community plugin quality and help people to skill up if necessary

Reply:
I can provide one later.
I tried to add the command lint as it was failing without it. Linting happens through prettier in my case, but in order to access to the executable, I need to pass through pnpm, like

```
pnpm prettier --write .
```

When it runs, ofc, it doesn't show pnpm.

I tried to add it through the right Gh action, but it seems to not be compatible with the "uses" 🤷‍♂️

Reply:
@Eomm This is the error that happened on the first run:

```sh
> fastify-passkit-webservice@1.4.0 test
> pnpm build && pnpm test:prepare && node --test && pnpm test:cleanup

sh: 1: pnpm: not found
```

Reply:
Yeah, got it - it would work if the `pnpm` dep would be added to the package.json deps or in a `postinstall` script.

Nevermind

Reply:
Thank you anyway, @Eomm! I think I'll take inspiration from that workflow